src/fu-binder-cli-bridge.cpp

```
@@ -0,0 +1,372 @@
+
+/* define the listener object */
+class FwupdEventListenerImpl : public aidl_fwupd::BnFwupdEventListener
+{
+      public:
+	::ndk::ScopedAStatus
+	onChanged() override
+	{
+		g_debug("engine state changed");
+		return ::ndk::ScopedAStatus::ok();
+	}
+
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If G_END_DECLS is a macro then please configure it.

(unknownMacro)